src/olympia/promoted/models.py

```diff
@@ -58,23 +58,23 @@ def all_applications(self):
 
     @property
     def approved_applications(self):
-        """The applications that the current promoted group is approved for."""
+        """The applications that the current promoted group is approved for.
+        Only listed versions are considered."""
```

Comment:
This might seem like a problem now we have unlisted versions for Notable, but for our purposes - `due_date`s and auto approval - it works just fine, because we don't consider if the Addon is "approved" for the promoted group, just if it's in the group.  

(Having unlisted versions changed the approved for Notable status too brought too many problems - chiefly that if an unlisted version had an approval but the current (listed) version didn't we'd be indicating that the listed versions of the add-on were human reviewed and okay based on the unlisted versions which could be very different)